tests/upgrade/test_crossgrade.sh

```diff
@@ -365,19 +370,19 @@ resetCluster() {
     echo_and_run_or_die kubectl label namespace "${TEST_NAMESPACE}" istio-injection=enabled
 }
 
-# Returns 0 if the passed string has form "Code 200 : 6601 (94.6 %)" and the percentage is smaller than ${MIN_200_PCT_FOR_PASS}
-percent200sAbove() {
-    local s=$1
-    local regex="Code 200 : [0-9]+ \\(([0-9]+)\\.[0-9]+ %\\)"
-    if [[ $s =~ $regex ]]; then
-        local pct200s="${BASH_REMATCH[1]}"
-        if (( pct200s > MIN_200_PCT_FOR_PASS )); then
-            return 0
-        fi
-        return 1
-    fi
-    echo "No Code 200 percentage found in log."
-    return 1
+# Return 1 if the specific error code percentage exceed corresponding threshold
+errorPercentBelow() {
```

Comment:
It would boost the readability of this function a lot if we would use local variables with good names for the parameters, e.g.

```
local LOG=$1
local SEARTERM=$2
local LIMIT=$3
```

Reply:
added local variables